config/validation.ts

```diff
@@ -0,0 +1,7 @@
+import Joi from 'joi';
+
+export const validationSchema = Joi.object({
+  values: {
+    PORT: Joi.number().required(),
```

Comment:
```suggestion
    PORT: Joi.number().default(8080),
```

аби уникнути додаткових перевірок